package io.kyberorg.yalsee.exception;

import io.kyberorg.yalsee.ui.err.raw.RawServerErrorPage;

/**
 * Very general exception, which not intended to be thrown, just {@link RawServerErrorPage} handles it.
 *
 * @since 3.11
 */
public class RawServerException extends YalseeException {
}
